discord/soundboard.py

```diff
@@ -0,0 +1,271 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-2021 Rapptz
```

Comment:
If the file contains no code from another source, we don't have to state another copyright.
```suggestion
```